test/component/actions_test.js

```diff
@@ -275,7 +275,7 @@ describe('Actions', function() {
                     clients.PostAction(action, function(error, data) {
                         should.not.exist(error);
                         data.should.have.property('statusCode', 200);
-                        setTimeout(callback, 500);
+                        setTimeout(callback, 1000);
```

Comment:
Not sure what this timeout is but if "1000" means a time, probably is better to have it unhardwired.